### ISSUE TITLE
`game.sounds.setVolume` throws error

### ISSUE THREAD

**Issue by aquisenberry** [SplatJS/splat-ecs#26]
In sound-loader.js:

```
SoundLoader.prototype.setVolume = function(gain) {
    this.volume = gain;
    this.gainNode.gain  = gain;
    this.muted = false;
};
```

this.gainNode.gain = gain; is throwing the following error when calling game.sounds.setVolume(num):

> Cannot set property gain of #<GainNode> which has only a getter

**Comment by ericlathrop** [comment]
Looking at how the rest of the code uses `gainNode`, it's probably just missing the `.value`: `this.gainNode.gain.value = this.volume;`

Can you try this, and if it works, send a PR? Thanks!

**Comment by ericlathrop** [comment]
Fixed with PR#40
